awsgi/__init__.py

```diff
@@ -16,29 +17,49 @@ def convert_str(s):
         return s
 
 
-def response(app, event, context):
-    sr = StartResponse()
+def convert_b46(s):
```

Comment:
This variable name could be more descriptive. I assume `s` stands for string but it could be clearer.

Reply:
Yes I see what you mean, however I was following the pattern for the existing `convert_str` functions https://github.com/SteadBytes/awsgi/blob/8316b655b8f4e424c2511d17bdb7aae0e44b674b/awsgi/__init__.py#L9 . Is it worth changing those as well? Would `string` be better?

Reply:
Geez. Sorry I was looking at narrow context. I think `s` is OK for consistency, but something more descriptive is OK too. I wouldn't use `string` because it looks like a reserved word or type.

Reply:
Haha no worries! Good point about `string` being potentially confusing, I imagine this would be worth revisiting if this module gets extended further in future :+1:  Thanks for merging!